grammars/css.cson

```diff
@@ -20,294 +20,1205 @@
+          |lavender|lavenderblush|lawngreen|lemonchiffon|lightblue|lightcoral|lightcyan|lightgoldenrodyellow
+          |lightgray|lightgreen|lightgrey|lightpink|lightsalmon|lightseagreen|lightskyblue|lightslategray
+          |lightslategrey|lightsteelblue|lightyellow|limegreen|linen|magenta|mediumaquamarine|mediumblue
+          |mediumorchid|mediumpurple|mediumseagreen|mediumslateblue|mediumspringgreen|mediumturquoise
+          |mediumvioletred|midnightblue|mintcream|mistyrose|moccasin|navajowhite|oldlace|olivedrab|orangered
+          |orchid|palegoldenrod|palegreen|paleturquoise|palevioletred|papayawhip|peachpuff|peru|pink|plum
+          |powderblue|rebeccapurple|rosybrown|royalblue|saddlebrown|salmon|sandybrown|seagreen|seashell
+          |sienna|skyblue|slateblue|slategray|slategrey|snow|springgreen|steelblue|tan|thistle|tomato
+          |transparent|turquoise|violet|wheat|whitesmoke|yellowgreen)
+          (?![\\w-])
+        '''
+        'name': 'support.constant.color.w3c-extended-color-name.css'
+      }
+      {
+        # Current text colour
+        'match': '(?i)(?<![\\w-])currentColor(?![\\w-])'
+        'name': 'support.constant.color.current.css'
+      }
+      {
+        # These colours are deprecated in CSS3: http://www.w3.org/TR/css3-color/#css2-system
+        'match': '''(?xi) (?<![\\w-])
+          (ActiveBorder|ActiveCaption|AppWorkspace|Background|ButtonFace|ButtonHighlight|ButtonShadow
+          |ButtonText|CaptionText|GrayText|Highlight|HighlightText|InactiveBorder|InactiveCaption
+          |InactiveCaptionText|InfoBackground|InfoText|Menu|MenuText|Scrollbar|ThreeDDarkShadow
+          |ThreeDFace|ThreeDHighlight|ThreeDLightShadow|ThreeDShadow|Window|WindowFrame|WindowText)
+          (?![\\w-])
+        '''
+        'name': 'invalid.deprecated.color.system.css'
+      }
+    ]
+  'combinators':
+    'patterns': [
+      {
+        'match': '/deep/|>>>'
+        'name': 'invalid.deprecated.combinator.css'
+      }
+      {
+        'match': '>|\\+|~|>>'
+        'name': 'keyword.operator.combinator.css'
+      }
+    ]
+  'commas':
+    'match': ','
+    'name': 'punctuation.separator.list.comma.css'
+  'comment-block':
+    'begin': '/\\*'
+    'beginCaptures':
       '0':
-        'name': 'keyword.control.at-rule.charset.css'
-      '1':
-        'name': 'punctuation.definition.keyword.css'
-    'end': '\\s*((?=;|$))'
-    'name': 'meta.at-rule.charset.css'
+        'name': 'punctuation.definition.comment.begin.css'
+    'end': '\\*/'
+    'endCaptures':
+      '0':
+        'name': 'punctuation.definition.comment.end.css'
+    'name': 'comment.block.css'
+  'escapes':
     'patterns': [
       {
-        'include': '#string-double'
+        'match': '\\\\[0-9a-fA-F]{1,6}'
+        'name': 'constant.character.escape.codepoint.css'
       }
       {
-        'include': '#string-single'
+        'match': '\\\\$\\n?'
```

Comment:
Yeah. It's a TextMate hack for advancing the pointer past the end-of-line mark. A `begin`/`end` pattern which stops at `$` will continue to span lines if `$\n?` matches.